src/main/resources/log4j.properties

```diff
@@ -1,8 +1,15 @@
-# variables are substituted from filters-[production|development].properties
-log4j.rootLogger=INFO, CONSOLE
-#log4j.rootLogger=DEBUG, CONSOLE
+status = info
```

Comment:
I think this file has to be named log4j**2**.properties as we are using log4j2 logger. Not sure if log4j works too

Reply:
Fixed.